scripts/npm/link/index.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var path = require( 'path' );
+var fs = require( 'fs' );
+var componentsDir = path.join( __dirname, '..', '..', '..', 'components' );
+var packageContent = fs.readFileSync( path.join( __dirname, '..', '..', '..', 'package.json' ) );
+var packageJSON = JSON.parse( packageContent );
+var exec = require( 'child-process-promise' ).exec;
+
+function filterComponents( deps ) {
+  for ( var component in deps ) {
+    if ( /^cf-/.test( component ) ) {
```

Comment:
Right, but if you start searching NPM, you might think you need `cf-theme-cfpb` when you actually don't. The same could happen to future `cf-util` packages. I think we ought to prefix with `cfpb-cf-` for bureau specific packages.